src/scripts/fetch/fetch.ts

```diff
@@ -0,0 +1,58 @@
+interface Fetchparams {
+	param: string;
+	value: any;
+}
+
+/**
+ * Fetches data from themoviedb.org API endpoint and returns to the user
+ *
+ * @param URLExtension - the extension over the base API for example: /movie/550
+ * @param APIkey - your api key
+ * @param URLparams array of url parameters to add to fetch request. For example [{param:Query, value:string}]
+ * @returns Returns base object from the API parsed through response.json
+ */
+export const APIFetch = async (
+	URLExtension: string,
+	APIkey: string,
+	URLparams?: Fetchparams[]
+) => {
+	//Basic Fetch URL without params
+	const url = `https://api.themoviedb.org/3${URLExtension}?api_key=${APIkey}`;

```

Comment:
Could we use API key from .env directly here instead of passing as a parameter so that each time we use APIfetch we don't have to pass key everytime

Reply:
We could. I intentionally left it as a param in case people named their api key variable differently, but I can just pass it through if we set a standard.